Makefile

```diff
@@ -16,6 +16,7 @@ license: $(TOOLBIN)/license_finder
 .PHONY: generate
 generate: $(TOOLBIN)/controller-gen $(TOOLBIN)/json-schema-generator
 	$(TOOLBIN)/json-schema-generator -r ./manager/apis/app/v1alpha1/ -o charts/fybrik/files/taxonomy/
+	$(TOOLBIN)/json-schema-generator -r ./manager/apis/app/v1beta1/ -o charts/fybrik/files/taxonomy/
```

Comment:
Why is the old one used for generating taxonomy files? There will be only one version of the file in taxonomy.

Reply:
In this case, yes, the old version can be removed. In the future, if APIVersion's schemas can be different, we will have hard time with taxonomy.  
Removed.